portfolio/src/main/java/com/google/sps/data/CalendarEvent.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+
+/** Models a calendar event. It can be used for both creation and import flow. */
+public class CalendarEvent {
+  private String name;
+  private final String startTimeString;
+  private final String endTimeString;
+
+  // TODO(hollyyuqizheng): Add an ID field if necessary.
+
+  public CalendarEvent(String name, String startTimeString, String endTimeString) {
```

Comment:
Let's add some comment. And it should describe what startTimeString and endTimeString look like.

Reply:
Updated.